Automattic/jetpack
# VideoPress: add plan support checks for all plans.

Fixes #10085

#### Changes proposed in this Pull Request:

Make sure that all users of a Premium or Business plan, whether it's a one-year or 2-year plan, can use the VideoPress feature.

#### Testing instructions:

1. Visit the wp-admin panel of an AT site with a one-year Business Plan
2. Go to Jetpack > Settings
3. Under Media > Video, you should be able to see a tab with the text `Enable high-speed, ad-free video player` to activate or deactivate VideoPress.
4. Replace the site's Business Plan to a two-year one in Store Admin
5. Go back to Jetpack > Settings
6. Under Media, VideoPress seems to be unavailable, and the tab to turn it on or off is missing.  Instead, there is a blue "Upgrade" button next to the text `Host fast, high-quality, ad-free video.`
7. Install the [Jetpack beta plugin](https://jetpack.com/download-jetpack-beta/)
8. Visit /wp-admin/admin.php?page=jetpack-beta and activate the `fix/videopress-plans` branch.
9. Go back to Jetpack settings and make sure you can now activate the VideoPress feature.


#### Proposed changelog entry for your changes:
- VideoPress: do not block access to the Video settings for 2-year plans.
